CHANGES.txt

```diff
@@ -1,6 +1,14 @@
 Changes
 =======
 
+
+1.0rc4 (?)
+-------------------
+New features:
+
+- CRS.epsg property added to determine the epsg code from the projection if possible (#1288).
+
+
```

Comment:
@snowman2 will you put this new feature under 1.0rc3? Also, I will recategorize the absence of `to_epsg` as a bug, it's something that should be there.

Reply:
Sure.